PolicyProject1/src/main/java/com/example/demo/model/Policy.java

```diff
@@ -1,84 +1,69 @@
 package com.example.demo.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import java.time.LocalDateTime;
 
+/**
+ * this class uses lombok annotations.
+ * Refer: https://www.baeldung.com/lombok-ide for more info
+ */
+@Data
+@NoArgsConstructor

```

Comment:
@NoArgsConstructor annotation to  used tp generate the default constructor that takes no arguments.